dist/css/tooltip.css

```diff
@@ -0,0 +1,33 @@
+@-webkit-keyframes fadeOut {
+  from {
+    opacity: 1; }
+  to {
+    opacity: 0; } }
+
+@keyframes fadeOut {
+  from {
+    opacity: 1; }
+  to {
+    opacity: 0; } }
+
+.tooltip-container {
```

Comment:
Pls rename this class with prefix like `.just-validate-tooltip-container`
Other classes should rename too